examples/LaTeX/curvi_linear_latex.py

```diff
@@ -92,6 +96,8 @@ def derivatives_in_prolate_spheroidal_coordinates():
     A = ps3d.mv('A','vector',f=True)
     B = ps3d.mv('B','bivector',f=True)
 
+    print('#Derivatives in Prolate Spheroidal Coordinates')
+
```

Comment:
Mind making a new PR with just the changes to this file (and the notebook changes entailed)? I remember being confused by the output without these dividers.

Reply:
OK

Reply:
I'll have to do that even later tonight (need a full dev environment back home)

Reply:
No need to do this any time soon, just might be nice to  pick up at some point.